mobify/platform-scaffold
# Setup to use the new analytic connector

**JIRA**: https://mobify.atlassian.net/browse/MTT-602
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/626

## Changes
- Setup required to use the new Analytic Connector

## How to test-drive this PR

**Requirement** We need the progressive-web-sdk's **develop** branch

For whatever reason, when you `npm link` the `progressive-web-sdk`, the react app will crash at PDP.

We need to get to the checkout in order test transaction analytics.

1. Change `package.json` so that progressive SDK is pointing to your local file `"progressive-web-sdk": "file:../../progressive-web-sdk"`
2. Run `rm -r node_modules/progressive-web-sdk/ && npm i && cd node_modules/progressive-web-sdk/ && npm i && npm run dev:build && rm -r node_modules/react node_modules/react-dom && cd -` This builds the SDK in your local node_modules
3. Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)

Observe the console log:
- Navigate normally and you see pageview analytics
- Buy a product and you will see transaction analytics
